requirements-dev.txt

```diff
@@ -5,3 +5,4 @@ sure<1.2.4
 coverage
 freezegun
 flask
+boto3
```

Comment:
Was passing travis due to https://github.com/spulec/moto/commit/491df739d6ac864b9aa236cbc376806ee8a116ed